### DISCUSSION TITLE
Consider how map sub-category controls could be separated

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1521] [Ideas]
Currently the chart/table controls site outside of the actual tab. This is currently ok as controls for chart and table are the same.

However, if we want to add map controls (which would be different), then the data controls will need to be moved inside the tabs so it is clear that the controls and visualisation are connected

**Comment by LucyGwilliamAdmin** [comment]
@phillipgardiner have I documented this correctly?

**Reply by phillipgardiner** [reply]
@LucyGwilliamAdmin In essence yes. However I've not been in the loop recently, but this approach was 'a' option to look into rather than a definite solution. I think this ultimately ties into what the needs within maps are and the subsequent priority of maps.
